### PR TITLE
Update Event watcher docs

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
I realized using `env` as a label is more general, so let me do tiny updates.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/user-guide/event-watcher.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ The full list of configurable `EventWatcher` fields are [here](/docs/user-guide/
 To register a new value corresponding to Event such as the above in the control-plane, you need to perform `pipectl`.
 And we highly recommend integrating a step for that into your CI workflow.
 
-According to [this guide](/docs/user-guide/command-line-tool/#installation), you first install it on your CI system or where you want to run.
-And then you grab the API key for the control-plane according to [this guide](/docs/user-guide/command-line-tool/#authentication).
+You first need to set-up the `pipectl`:
+
+- Install it on your CI system or where you want to run according to [this guide](/docs/user-guide/command-line-tool/#installation).
+- And then you grab the API key to which the `READ_WRITE` role is attached according to [this guide](/docs/user-guide/command-line-tool/#authentication).
```
Comment (khanhtc1202) [inline docs/content/en/docs/user-guide/event-watcher.md:56]: ~And then you~

File: docs/content/en/docs/user-guide/event-watcher.md [hunk 2]
```diff
@@ -119,21 +123,21 @@ spec:
...
-        repoId: repo-1
+        env: dev
         appName: helloworld
       replacements:
         - file: helloworld/.pipe.yaml
           yamlField: $.spec.input.helmChart.version
 ```
 
-Push a new version `0.2.0` as a data.
+Push a new version `0.2.0` as a data when the Helm release is complete.
```
Comment (khanhtc1202) [inline docs/content/en/docs/user-guide/event-watcher.md:133]: `is completed`

Comment (nakabonne) [reply docs/content/en/docs/user-guide/event-watcher.md]: Thanks.

**Comment by nakabonne** [comment]
PTAL again.

File: docs/content/en/docs/user-guide/event-watcher.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/docs/user-guide/event-watcher.md:56]: @nakabonne I think you could remove the part `And then you` here, due to the bullet points pattern 😄

Comment (nakabonne) [reply docs/content/en/docs/user-guide/event-watcher.md:56]: Ahh, that's what you mean, thank you!

**Pull request merged**